tools/Performance/DynamoPerformanceTests/ResultsComparer.cs

```diff
@@ -233,7 +275,8 @@ private List<BenchmarkComparison> CreateComparisons(string basePath, string newP
             foreach(var result in newResults.Values)
             {
                 var baseResult = GetBaseResult(result, baseResults);
-                comparisons.Add(new BenchmarkComparison(baseResult, result));
+                //NOTE(MJK) - to control determination of pass/fail conditions for each test, pass them here.
+                comparisons.Add(new BenchmarkComparison(baseResult, result, null));
```

Comment:
What would it take to handler `MeanDelta` set to e.g. 15? Would you provide an example?

Reply:
yes, it would look like
```
comparisons.Add(new BenchmarkComparison(baseResult, result, ()=>{
if(MeanDelta > 15)
{
return FAIL;
}
return PASS;
 });
 ```

do you want me to add this to the comments?

Reply:
Thx, please